CHANGELOG.md

```diff
@@ -1,14 +1,23 @@
 # Braintree iOS SDK Release Notes
 
 ## unreleased
+* Add the following properties to `BTVenmoRequest`
```

Comment:
General question, why are these fields being added? Is this a feature or just so merchants can send additional details in the request?

Reply:
Typically we'd put this under a section with the module name like:
```
* BraintreeVenmo
    * heading and properties nested here 
```

Reply:
Its a feature. Merchants can add pass these fields and based on that, new info (amounts and line items) will be displayed on the Venmo Paysheet, as well as collection of shipping and billing addresses.

I'll update the CHANGELOG file.